.github/workflows/eaa_kbc.yml

```diff
@@ -25,34 +25,47 @@ jobs:
     steps:
       - name: Install rats-tls dependencies
         run: |
-          apt-get update && apt-get install -y autoconf gcc g++ make wget git cmake \
+          sudo apt-get update;
+          sudo apt-get install -y autoconf gcc g++ make wget git cmake \
           libseccomp-dev binutils-dev libprotoc-dev protobuf-compiler \
           pkg-config libssl-dev openssl libtool file curl gnupg;
 
-          wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz &&\
-          tar -zxvf v1.3.1.tar.gz && cd protobuf-c-1.3.1 && ./autogen.sh && \
-          ./configure && make && make install
+          sudo wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz;
+          sudo tar -zxvf v1.3.1.tar.gz;
```

Comment:
Do you have to use this version? If so:

- Please add a comment explaining why.
- Please use a variable to avoid the duplicated version string.

... but if you just need the latest, you could use something like:

```bash
local latest
latest=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/protobuf-c/protobuf-c/releases/latest)
```